src/globals.h

```diff
@@ -1674,6 +1674,19 @@ EXTERN int	cmdwin_result INIT(= 0); // result of cmdline window or 0
 
 EXTERN char_u no_lines_msg[]	INIT(= N_("--No lines in buffer--"));
 
+EXTERN char typename_unknown[]	INIT(= N_("unknown"));
+EXTERN char typename_int[]	INIT(= N_("int"));
+EXTERN char typename_longint[]	INIT(= N_("long int"));
+EXTERN char typename_longlongint[]	INIT(= N_("long long int"));
+EXTERN char typename_unsignedint[]	INIT(= N_("unsigned int"));
```

Comment:
I'm not sure if the format specifier type needs to be translated in the error message. Types of the C language are most often written int, long, etc.